### DISCUSSION TITLE
Join

### DISCUSSION THREAD

**Discussion by Secret-chest** [ScratchAddons/ScratchAddons#2550] [Help] [answered]
Can I join the org? I know new members are rarely added but I would still like to join

**Comment by TheColaber** [comment, marked as answer]
I'm sure you've seen #2537, So being patient and contributing, like I and many of the existing org members have done, will bring you to the team.

**Reply by WorldLanguages** [reply]
no it won't

**Reply by Secret-chest** [reply]
I will contribute more. Thanks

**Reply by WorldLanguages** [reply]
TheColaber made it sound like it's guaranteed you get invited if you're active but that's not the case 😭
